locales/pt.json

```diff
@@ -89,127 +89,127 @@
- 	"no contracts found": "no contracts found",
- 	"Blockchain component is disabled in the config": "Blockchain component is disabled in the config",
- 	"Couldn't connect to an Ethereum node are you sure it's on?": "Couldn't connect to an Ethereum node are you sure it's on?",
- 	"make sure you have an Ethereum node or simulator running. e.g '%s'": "make sure you have an Ethereum node or simulator running. e.g '%s'",
- 	"executing": "executing",
- 	"the transaction was rejected; this usually happens due to a throw or a require, it can also happen due to an invalid operation": "the transaction was rejected; this usually happens due to a throw or a require, it can also happen due to an invalid operation",
- 	"Cannot find file %s Please ensure you are running this command inside the Dapp folder": "Cannot find file %s Please ensure you are running this command inside the Dapp folder",
- 	"no config file found at %s using default config": "no config file found at %s using default config",
- 	"HTTP contract file not found": "HTTP contract file not found",
- 	"contract file not found": "contract file not found",
- 	"file not found, creating it...": "file not found, creating it...",
- 	"No Blockchain node found": "No Blockchain node found",
- 	"Ethereum node (version unknown)": "Ethereum node (version unknown)",
- 	"this event is deprecated and will be removed in future versions %s": "this event is deprecated and will be removed in future versions %s",
- 	"Error while downloading the file": "Error while downloading the file",
- 	"Plugin {{name}} can only be loaded in the context of \"{{contextes}}\"": "Plugin {{name}} can only be loaded in the context of \"{{contextes}}\"",
- 	"error running service check": "error running service check",
+ 	"Init complete": "Inicialização completa",
+ 	"App ready at ": "Aplicativo disponivel em ",
+ 	"Next steps:": "Próximos passos:",
+ 	"open another console in the same directory and run": "abra outro console no mesmo ficheiro e execute",
+ 	"For more info go to http://embark.status.im": "Para mais informações, vá para http://embark.status.im",
+ 	"%s : instanceOf is set to itself": "%s : instanceOf atrelado a si mesmo",
+ 	"{{className}} has code associated to it but it's configured as an instanceOf {{parentContractName}}": "{{className}} tem código associado à ele, porém está configuraco como instanceOf {{parentContractName}}",
+ 	"Error Compiling/Building contracts: ": "Erro ao Compilar/Construir contratos: ",
+ 	"Error: ": "Erro: ",
+ 	"there are two or more contracts that depend on each other in a cyclic manner": "existem dois ou mais contratos que dependem um do outro de um jeito cíclico",
+ 	"Embark couldn't determine which one to deploy first": "Embark não pôde determinar qual deverá ser publicado primeiro",
+ 	"{{inputName}} has not been defined for {{className}} constructor": "{{inputName}} não foi definido para o contrustor de {{className}}",
+ 	"error deploying %s": "erro ao mostrar %s",
```

Comment:
`mostrar` is not the same as `deploying`.

Reply:
Oops, not sure how I managed to write ```mostrar```there D: